components/api-invocation-hanlder/pom.xml

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>component-dep</artifactId>
+        <groupId>com.wso2telco.dep</groupId>
+        <version>2.2.8-SNAPSHOT</version>
+        <relativePath>../../pom.xml</relativePath>
+    </parent>
+
+    <groupId>com.wso2telco.dep</groupId>
+    <artifactId>api-invocation-handler</artifactId>
+    <version>1.0.0-SNAPSHOT</version>
```

Comment:
Avoid using separate versions inside components. Allow to inherit from parent

Reply:
Removed from POM